x-pack/plugins/infra/server/routes/services/index.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  GetServicesRequestQueryRT,
+  GetServicesRequestQuery,
+  ServicesAPIResponseRT,
+} from '../../../common/http_api/host_details';
+import { InfraBackendLibs } from '../../lib/infra_types';
+import { getServices } from '../../lib/host_details/get_services';
+import { validateStringAssetFilters } from './lib/utils';
+import { createSearchClient } from '../../lib/create_search_client';
+import { buildRouteValidationWithExcess } from '../../utils/route_validation';
+
+export const initServicesRoute = (libs: InfraBackendLibs) => {
+  const { framework } = libs;
+  framework.registerRoute<unknown, GetServicesRequestQuery, unknown>(
+    {
+      method: 'get',
+      path: '/api/infra/services',
```

Comment:
Shouldn't we make this API `internal`? Otherwise, we should version it, shouldn't we?

Reply:
After discussion, this we'll be addressed here: https://github.com/elastic/kibana/issues/175975

Reply:
The router config will default to internal.  We should probably look at our paths and figure out if we need to change them, but Logs and Infra currently use the "/api/infra" for all the endpoints with no "internal".  I'd like to address this in a separate issue as its easy enough to change.